nms/packages/magmalte/app/state/lte/SubscriberState.js

```diff
@@ -289,6 +289,7 @@ export type SubscriberQueryType = {
   setTokenList: (Array<string>) => void,
   ctx: SubscriberContextType,
   subscriberMetrics?: {[string]: Metrics},
+  deleteTable?: boolean,
```

Comment:
Add some comments for this type

Reply:
Don't have this nullable
```suggestion
  deleteTable: boolean,
```